wiki/Article_Style_Guide/en.md

```diff
@@ -181,67 +279,251 @@ For example, use:
 
-For example, instead of `Do NOT contact peppy for ingame issues!` use `Do **not** contact peppy for ingame issues!`
+All lists (bulleted or numbered) **must** have one empty line space before the list starts.
+Otherwise the list may be parsed incorrectly.
+
+```
+## Section
+
+- item A
+- item B
+```
+
+To break lines in a list, use:
+
+```
+1. The combo fire was a gameplay feature.
+   It would display a burning yellow flame after obtaining a combo of 30.
+   - This was disabled on March 5, 2013.
+     Most likely due to performance concerns.
+```
+
+There are two kinds of lists:
+
+- bulleted
+  - used when the order of each item does not matter (e.g. describing an object)
+- numbered
+  - used when the order of each item does matter (e.g. tutorial instuctions)
+
+### Sub-Lists
+
+Sub-lists are lists that are indented underneath a listed item.
+
+Please limit to a level of four (4) sub-lists at a time (if you need more, you should reconsider about how you are structuring the list).
+
+```
+- item A
+  - sub-item A.1
+    - sub-sub-item A.1.i
+      - sub-sub-sub-item A.1.i.a
+      - sub-sub-sub-item A.1.i.b
+    - sub-sub-item A.1.ii
+      - sub-sub-sub-item A.1.ii.a
+    - sub-sub-item A.1.iii
+  - sub-item A.2
+    - sub-sub-item A.2.i
+  - sub-item A.3
+- item B
+```
 
 ## Code
 
 Use code (inline) when noting:
 
 - a key on the keyboard
-  - i.e. `B`, `Alt` or `Ctrl` + `Shift` + `A`
-- a name of a button
-  - i.e. `1. Let's do it.` or `4. Noooo! I didn't mean to!`
-- a name of a folder
-  - i.e. `Exports` or `Downloads`
-- a name of a directory
-  - i.e. `/Songs/` or `/Skins/User/`
-- a name of a keyword
-  - i.e. `ComboBurstStyle` or `[Events]`
+  - should use capital letters for single characters and [camel case](https://en.wikipedia.org/wiki/Camel_case) for modifers.
+  - examples:
+    - `.`
+    - `Alt`
+    - `Ctrl` + `Shift` + `A`
+- the text of a button
+  - examples:
+    - `1. Let's do it.`
+    - `4. Noooo! I didn't mean to!`
+- the name of a folder
+  - examples:
+    - `Exports`
+    - `Downloads`
+- the name of a directory
+  - examples:
+    - `/Songs/`
+    - `/Skins/User/`
+- the name of a keyword
+  - examples:
+    - `ComboBurstStyle`
+    - `[Events]`
 - the name of a file extension
-  - i.e. `.jpg` or `.osz`
+  - must add a period (`.`) then the file extension in lowercase letters.
+  - examples:
+    - `.jpg`
+    - `.gif`
+    - `.osz`
+- the name of a chat channel
+  - examples:
+    - `#lobby`
+    - `#osu`
+
+### Code Blocks
 
-When noting a keyboard key, use capital letters for single characters and [Camel case](https://en.wikipedia.org/wiki/Camel_case) for modifers.
+When using code blocks, use the ` ``` ` (triple grave mark) syntax.
 
-When naming a file extension, add a period (`.`) then the file extension in lower case letters.
+The osu!wiki site has a feature where you _could_ use four spaces to trigger the code blocks; however, this is **heavily** discouraged.
 
 ## Links
```

Comment:
it can't be the long link; I had mentioned in the dev chat that it was the wiki container as benpigchu had just said... but I will note that all links must have link text (though it kinda has been stated already)